README.md

```diff
@@ -4,109 +4,133 @@
 
 <p>&nbsp;</p>
 
-# The Pact Programming Language
+# Pact: Smart Contract Programming Language
 
-[Pact](http://kadena.io/build) is an open-source, Turing-**in**complete smart contract language that has been purpose-built with blockchains first in mind. Pact focuses on facilitating transactional logic with the optimal mix of functionality in authorization, data management, and workflow.
+[Pact](http://kadena.io/build) is an open-source, smart contract language designed to execute transactional logic efficiently in a blockchain environment. 
+The Pact smart contract programming language is intentionally **Turing-incomplete** to prevent recursion and unbounded looping that could be exploited or disrupt the blockchain network. 
+The Pact language focuses on providing smart contract authors with the right set of features to optimize security, performance, and transparency while providing flexible methods for authorizing access to contract data and function, managing data storage, and designing application workflows.
 
-Read the whitepaper:
+For a historical perspective on the design of Pact, see [The Pact Smart Contract Language](https://d31d887a-c1e0-47c2-aa51-c69f9f998b07.filesusr.com/ugd/86a16f_442a542b64554cb2a4c1ae7f528ce4c3.pdf) whitepaper.
```

Comment:
That link doesn't look good tbh. Can we host the white paper on the kadena.io site?

Reply:
@emilypi

Reply:
The whitepaper is available from the main website, but the link is similar:
https://cdn.sanity.io/files/agrhq0bu/production/70f01649395af96655ca94d331fb1bd01af9fc8a.pdf
Given that that doc is really old (2016-2017), I changed the link to point to the whitepapers page instead of the PDF. If people are interested in reading the whitepaper, they can download the PDF from there.

Reply:
Cant we upload it to kadena.io ?